doc/source/starting.rst

```diff
@@ -33,32 +33,39 @@ System Wide (for the current user)
+   You may need to run ``pip install --user montepy`` if you are not allowed to install the package.
 
 Install specific version for a project
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 The best way maybe to setup a project-specific `conda <https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html>`_, 
 `Mamba <https://mamba.readthedocs.io/en/latest/user_guide/concepts.html>`_, 
 or a `venv <https://docs.python.org/3/library/venv.html>`_ environment.
 The steps for installing inside one of those environments are the same as the previous steps.
+You can specify a specific version from `PyPI`_ be installed using:
+
+``pip install montepy==<version>``
 
-Another option is to clone the repository and to use symbolic-links. In this scenario we'll assume that your local
-repository is located at ``~/dev/montepy``, and your project is located at ``~/foo/bar``. 
 
-#. Move to the repository parent folder: ``cd ~/dev``
-#. Clone this repository: ``git clone https://github.com/idaholab/montepy.git`` 
-#. Enter the repository: ``cd montepy``
-#. Checkout the specific version you want. These are tagged with git tags
+Best Practices
+--------------
+
+Before we begin here are some guidelines to keep in mind while scripting your work with MCNP models.
 
-    #. You can list all tags with ``git tag``
+#. *Always* version control your input files (not output files) with `git <https://git-scm.com/>`_ or another tool.
+   If you are working with very large input models, like `the ITER model <https://doi.org/10.1038/s41560-020-00753-x>`_ you may want to consider `git-lfs <https://git-lfs.com/>`_.
 
-    #. You can then checkout that tag: ``git checkout <tag>``
+   #. Do learn some `git best practices <https://sethrobertson.github.io/GitBestPractices/>`_. "Update" is not a useful commit message.
 
-#. Install the dependent requirements: ``pip install -r requirements/common.txt``
-#. Move to your project folder: ``cd ~/foo/bar``
-#. Create a symbolic link in the project folder to the repository: ``ln -s ~/dev/montepy/montepy montepy``
+#. *Always* have backups. Don't be that person that loses the last months of work when your laptop falls in a pond. 
+   Make sure there's a cloud backup (could be OneDrive, GitHub, etc.). 
+   Just make sure you comply with any applicable corporate policies. 
 
-Now when you run a python script in that folder (*and only in that folder*) ``import montepy`` will use the specific version you want. 
+#. Don't overwrite your original file. Generally your script should open file "A", modify it, and then save it to file "B".
+   This way if there is a bug in your script you can debug it and rerun it because "A" still exists.
+   Also if down the road you need to make changes you can modify your script and rerun it. 
+   This is especially true if your script ever becomes qualified under an `ASME NQA-1 <https://en.wikipedia.org/wiki/ASME_NQA>`_ compliant Software Quality Assurance program,
```

Comment:
add punctuation. suggestion:

> Don't overwrite your original file. Generally, your script should: open file "A", modify it, and then save it to file "B".    This way, when there is a bug in your script, you can debug it and rerun it because "A" still exists. Later, if you need to make changes, you can modify your script and rerun it...